docs/projects/cn0561/index.rst

```diff
@@ -0,0 +1,280 @@
+
+=============== === ========== ===========
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_cn0561_dma  13  57         89
+spi_cn0561      12  56         88
+=============== === ========== ===========
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here </>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0561/zed
+   user@analog:~/hdl/projects/cn0561/zed$ make
+
+or
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0561/coraz7s
+   user@analog:~/hdl/projects/cn0561/coraz7s$ make
+
+or
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0561/de10nano
+   user@analog:~/hdl/projects/cn0561/de10nano$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheet: :adi:`AD4134`
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`CN0561 HDL project source code <projects/cn0561>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_clkgen`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac`
+     - :ref:`here <axi_dmac>`
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine`
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution`
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect`
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload`
+     - :ref:`here <spi_engine offload>`
+
+-  :ref:`SPI Engine Framework documentation <spi_engine>`
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**No-OS support**
+
+-  No-OS driver: :git-no-os:`AD4134/AD7134 driver <drivers/adc/ad713x>`
+
+-  No-OS project source code: :git-no-os:`CN0561 project <projects/cn0561>`
```

Comment:
Please use this format: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad7616_sdz/index.rst?plain=1#L381